Classes/Controller/BackendController.php

```diff
@@ -122,11 +154,17 @@ public function indexAction(NodeInterface $node = null)
             $this->redirectToUri($this->uriBuilder->uriFor('index', [], 'Login', 'Neos.Neos'));
         }
 
-        if ($node === null) {
-            $node = $this->findNodeToEdit();
+        if ($nodeAddress === null) {
+            $nodeAddress = $this->findNodeToEdit();
         }
 
-        $siteNode = $node->getContext()->getCurrentSiteNode();
+        $workspaceName = $this->userService->getPersonalWorkspaceName();
+        $workspace = $this->workspaceFinder->findOneByName(new WorkspaceName($workspaceName));
+        $subgraph = $this->contentGraph->getSubgraphByIdentifier($workspace->getCurrentContentStreamIdentifier(), $this->findDefaultDimensionSpacePoint());
+        $siteNode = $subgraph->findChildNodeConnectedThroughEdgeName($this->getRootNodeIdentifier(), new NodeName($this->siteFinder->findDefault()->nodeName));
```

Comment:
This seems wrong to me and only works when having a single site.

The actuals site node could be inferred from `$node` by traversing parents up to the static `sites` node.
Or we pass the site as an argument.
Or we include the site in something like a `NodeAddressWithSite` value object.

Reply:
totally true! will fix!